ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/cveListExceptionFlow.test.js

```diff
@@ -0,0 +1,90 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+import {
+    selectMultipleCvesForException,
+    selectSingleCveForException,
+    visitWorkloadCveOverview,
+} from './WorkloadCves.helpers';
+import { selectors } from './WorkloadCves.selectors';
+
+describe('Workload CVE List deferral and false positive flows', () => {
+    withAuth();
+
+    before(function () {
+        if (
+            !hasFeatureFlag('ROX_VULN_MGMT_WORKLOAD_CVES') ||
+            !hasFeatureFlag('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL')
+        ) {
+            this.skip();
+        }
+    });
+
+    beforeEach(() => {
+        // TODO - clean up any existing deferred or false positive CVEs
+    });
+
+    it('should disable multi-cve controls when no rows are selected', () => {
+        visitWorkloadCveOverview();
+        // Check that the select all checkbox is disabled
+        // Check that the bulk action menu is disabled
+        cy.get(selectors.tableRowSelectAllCheckbox).should('be.disabled');
+        cy.get(selectors.bulkActionMenuToggle).should('be.disabled');
```

Comment:
Corresponding assertions for disabled and enabled are a super pattern.

Especially because there have been isolated disabled assertions that had been obsolete for a long time.